README.md

```diff
@@ -130,19 +130,19 @@ Grafana setup. For a list of available plugins, see:
 ## Using it with the InfluxDB Community add-on
 
 Grafana does not come out of the box pre-configured, but letting it interact
-with the community InfluxDB add-on is pretty easy.
+with the community [InfluxDB add-on](hassio-addons/repository/addon-influxdb) is pretty easy.
 
-- Go to the InfluxDB admin interface and create a user for Grafana.
+- Follow the instructions from the [InfluxDB add-on](hassio-addons/repository/addon-influxdb#integrating-into-home-assistant) on how to intagrate into Home Assistant
 - Login into Grafana
 - Create a new datasource:
   - Name: Anything you want, e.g., Home Assistant
   - Type: InfluxDB
   - HTTP > URL: `http://a0d7b954-influxdb:8086`
   - HTTP > Access: Server (Default)
   - Auth: (leave them all disabled)
-  - InfluxDB Details > Database: _Your Home Assistant InfluxDB database_
-  - InfluxDB Details > User: _Grafana InfluxDB user_
-  - InfluxDB Details > Password: _Grafana InfluxDB password_
+  - InfluxDB Details > Database: _Your Home Assistant InfluxDB database_ e.g., `homeassistant`
+  - InfluxDB Details > User: _InfluxDB databaseuser_ e.g., `homeassistant`
```

Comment:
I do not agree on this since this does not follow: Every user should have their own credentials.
Since in this case, Grafana is a user/consumer of the InfluxDB data, it should have a different user than Home Assistant.

While your suggestion makes it easier for the user in forms of fewer actions, we should always guide a user into best practices and certainly not be dropping security.

Reply:
Agree and I'm doing it wrong on my installation. It's my first time using it.
Correct me, please. We use the same DB but create a different user/password for grafana?

Maybe we can add "create a user/password for granafa on your InfluxDB"?

Reply:
Yes, same database, different users. From a security perspective, you should have unique logins everywhere ;)

That is a pretty good suggestion 👍

Reply:
I can write it if needed.

Reply:
Yes please, go ahead 👍